sdk/common/src/main/java/io/opentelemetry/sdk/resources/Resource.java

```diff
@@ -120,7 +118,7 @@ private static String readVersion() {
       // we left the attribute empty
       return "unknown";
     }
-    return properties.getProperty("sdk.version");
+    return properties.getProperty("sdk.version", "unknown");
```

Comment:
This seems to match the behavior of line 119. Otherwise I could of course have the caller check null